src/main/java/uk/nhs/hee/tis/trainee/credentials/api/VerifyResource.java

```diff
@@ -68,10 +69,9 @@ ResponseEntity<String> verifyIdentity(@RequestHeader(HttpHeaders.AUTHORIZATION)
 
   @GetMapping("/callback")
   ResponseEntity<String> handleVerification(@RequestParam String code,
-      @ValidScope @RequestParam String scope,
       @UUID @RequestParam String state) {
     log.info("Received callback for credential verification.");
-    URI uri = service.completeCredentialVerification(code, scope, state);
+    URI uri = service.completeCredentialVerification(code, VERIFY_SCOPE, state);
```

Comment:
This was originally expected to be passed in so that we could verify any type of credential. While we don't do that right now it's possible we might in the future e.g. a GMC credential.